cliquet/events.py

```diff
@@ -0,0 +1,28 @@
+from cliquet.utils import strip_uri_prefix, Enum, current_service
+
+
+ACTIONS = Enum(CREATE='create',
+               DELETE='delete',
+               READ='read',
+               UPDATE='update')
+
+
+class ResourceChanged(object):
+    """Triggered when a resource is changed.
+    """
+    def __init__(self, action, resource, request):
+        self.request = request
+        service = current_service(request)
+        resource_id = service.viewset.get_name(resource.__class__) + '_id'
```

Comment:
With what @Natim suggested in #492 it becomes clear that we will want the resource name in the payload.

In the case of kinto, they are `bucket`, `group`, `collection`, `record`.

It can still be shipped as is, but if there is still some room for last-minute changes :)